src/components/Page/FolderPage.tsx

```diff
@@ -1,26 +1,27 @@
 import AddLinkForm from "../AddLinkForm";
-import '../../styles/Folder.css';
+// import '../../styles/Folder.css';
 import SearchBar from "../SearchBar";
 import NoLink from "../NoLink";
 import FolderLists from "../FolderLists";
 import Card from "../Card";
 import FolderTitle from "../FolderTitle";
-import {  useFolderPageFolder } from "../../hooks/folderPageHooks";
+import { useFolderPageFolder } from "../../hooks/folderPageHooks";
 import { useEffect, useState } from "react";
-import { ALL_FOLDER, FOLDER_PAGE_API_URL } from "../../constants/constants";
+import { ALL_FOLDER, FOLDER_API_URL } from "../../constants/constants";
 import axios from "axios";
+import { LinkType } from "../../constants/type";
 
-const linkURL = (forderId) => {
-  if(forderId === 0) return `${FOLDER_PAGE_API_URL}/1/links`;
-  return `${FOLDER_PAGE_API_URL}/1/links?folderId=${forderId}`;
+const linkURL = (forderId:number) => {
+  if(forderId === 0) return `${FOLDER_API_URL}/1/links`;
+  return `${FOLDER_API_URL}/1/links?folderId=${forderId}`;
 }
 
 const FolderPage = () => {
   const [activeFolder, setActiveFolder] = useState(0);
   const folders =[ALL_FOLDER, ...useFolderPageFolder()];
-  const [links, setLinks] = useState()
+  const [links, setLinks] = useState<LinkType[]>([])
 
-  const getLinkData = async (URL) => {
+  const getLinkData = async (URL:string) => {
```

Comment:
브라우저 (또는 브라우저의 JS엔진) 환경에서 제공하는 호스트객체인 [URL](https://developer.mozilla.org/ko/docs/Web/API/URL/URL)과 혼동이 있을 수 있는데, 사실 그냥 문자열 타입의 매개변수니까 소문자로 명시하는 편이 좋겠습니다
```suggestion
  const getLinkData = async (url :string) => {
```